python/ray/tests/test_output.py

```diff
@@ -0,0 +1,25 @@
+import re
+import pytest
+import ray
+
+
+def test_output(capsys, shutdown_only):
+    ray.init()
```

Comment:
Should we sleep a bit to avoid race conditions?

Reply:
Alternatively could call it in a subprocess

Reply:
subprocess could be better